### ISSUE TITLE
Brakujące źródła wtyczek

### ISSUE THREAD

**Issue by rysson** [CherryKodi/cherry-dev#29]
Takie kontrolne zgłoszenie. W CHERRY mamy 30 wtyczek. W cherry-dev 19. 
Gdzie są źródła brakujących wtyczek? Które z wtyczek są w naszym ręku (w sensie rozwoju)?

**Comment by rysson** [comment]
@notoco, @xulek, @podpis – ktoś z Was wie?

**Comment by xulek** [comment]
Wszystko co autorskie albo modyfikowane na potrzeby cherry jest w tym repo. Korzystając z okazji rysson jakbyś mógł to podaj mi swój email albo jakaś inną formę kontaktu abym mógł do ciebie prywatnie napisać.

Btw. Podpis gdzies przepadł, nie ma z nim kontaktu

**Comment by rysson** [comment]
Jasne, powinno być w commitach, spróbuj
```
git log -1 --author rysson
```

**Comment by rysson** [comment]
Wracając do tematu.
Czyli sport 356 nie jest pod naszą opieką?

**Comment by xulek** [comment]
Tą wtyczkę już przerabiało z 5 osób, każdy po swojemu :P

**Comment by rysson** [comment]
Tak, ale chodziło mi o źródła. To co postulowałem na początki i mamy cherry-dev. Ale ono nie obejmuje tego co oferujemy binarnie. 
Rozumiem, że trzeba przemyśleć podejście od strony prawa, więc nie naciskam.
